http-source/src/main/resources/application.yml

```diff
@@ -0,0 +1,8 @@
+server:
+  port: 0
+name: ${spring.application.name:http-source}
```

Comment:
we don't want to require module developers to put *anything* in the yml, so the real solution to the default bindings question needs to happen in the spring-cloud-stream code